java/core/src/test/java/co/worklytics/psoxy/rules/github/GitHubTests.java

```diff
@@ -647,7 +713,10 @@ public Stream<InvocationExample> getExamples() {
                 InvocationExample.of("https://api.github.com/repos/FAKE/REPO/pulls/42/commits", "pull_commits.json"),
                 InvocationExample.of("https://api.github.com/repos/FAKE/REPO/pulls/42/reviews", "pull_reviews.json"),
                 InvocationExample.of("https://api.github.com/repos/FAKE/REPO/pulls/42/reviews/10/comments", "pull_review_comments.json"),
-                InvocationExample.of("https://api.github.com/repos/FAKE/REPO/pulls/PULL_ID", "pull.json")
+                InvocationExample.of("https://api.github.com/repos/FAKE/REPO/pulls/PULL_ID", "pull.json"),
+                InvocationExample.of("https://api.github.com/users/p~IAUEqSLLtP3EjjkzslH-S1ULJZRLQnH9hT54jiI1gbN_fPDYrPH3aBnAoR5-ec6f?per_page=1", "user.json"),
+                InvocationExample.of("https://api.github.com/users/p~wD8RXfeJ5J-Po8ztEdwRQ-ae1xHBQKRMfNsFB5FFteZgtt4TBv84utnnumgFnjsR?per_page=1", "user.json"),
+                InvocationExample.of("https://api.github.com/users/p~IAUEqSLLtP3EjjkzslH-S1ULJZRLQnH9hT54jiI1gbN_fPDYrPH3aBnAoR5-ec6f", "user.json")
```

Comment:
BTW tried to setup the rule with a regex expression (see https://github.com/Worklytics/psoxy/pull/520/commits/23aed98c9a2e45a25065bb3aebba841b07354dcb) based on https://github.com/Worklytics/psoxy/blob/main/java/core/src/main/java/co/worklytics/psoxy/rules/msft/PrebuiltSanitizerRules.java#L28; have that working in tests but it was failing on end-to-end, so I've drop the regex and I put a pathTemplate instead.